src/System.Windows.Forms/src/System/Windows/Forms/Layout/TableLayout.cs

```diff
@@ -391,7 +388,7 @@ private static bool xAssignRowsAndColumns(ContainerInfo containerInfo, LayoutInf
                 }
 
                 //set the rowStart and columnStart to fixedElement's specifed position
```

Comment:
some typos.
```suggestion
                //set the rowStart and columnStart to fixedElement's specified position
```
And other parts of this file.



TableLayoutSettings.cs


``` c#
            Debug.Assert(!colSpecified || GetColumn(element) == column, "column position shoule equal to what we set");
            Debug.Assert(!rowSpecified || GetRow(element) == row, "row position shoule equal to what we set");
```

`shoule`